lib/phantom-script.js

```diff
@@ -69,8 +69,8 @@ page.open(skeleton, function (status) {
   page.paperSize = defaultFormat;
 
   if (args.runningsPath !== "nofile") {
-    var runningsArgs = JSON.parse(args.runningsArgs);
-    var runnings = require(args.runningsPath)(runningsArgs);
+    var runningsArgs = args.runningsArgs ? JSON.parse(args.runningsArgs) : undefined;
```

Comment:
This will not work.

When `options.runningsArgs` are null or undefined the `runningArgs` will be an empty string.

So this line will always try to parse `args.runningsArgs`, which will crash in case of an empty string.